deploy/mobile-client-crd.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: mobileclients.mobile.k8s.io
+  name: mobileclients.mobile.mdc.aerogear.org
```

Comment:
This value should be `mobileclients.mdc.aerogear.org`. Otherwise the following error message is given when trying to create the CRD:

_The CustomResourceDefinition "mobileclients.mobile.mdc.aerogear.org" is invalid: metadata.name: Invalid value: "mobileclients.mobile.mdc.aerogear.org": must be spec.names.pl
ural+"."+spec.group"._

Reply:
Thanks for the catch @craicoverflow @grdryn 
Pushed a new commit already